atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/util/ByteArrayUtilities.java

```diff
@@ -40,4 +42,21 @@ public static boolean areMapsEqual(Map<Cell, byte[]> map1, Map<Cell, byte[]> map
         }
         return true;
     }
+
+    public static boolean areRowResultsEqual(
+            NavigableMap<byte[], RowResult<byte[]>> first, NavigableMap<byte[], RowResult<byte[]>> second) {
+        if (first.size() != second.size()) {
+            return false;
+        }
+
+        for (Map.Entry<byte[], RowResult<byte[]>> e : first.entrySet()) {
+            if (!second.containsKey(e.getKey())) {
+                return false;
+            }
+            if (!e.getValue().equals(second.get(e.getKey()))) {
+                return false;
```

Comment:
Relying on RowResults.equals here makes sense. But would still be good to explicitly test this util.